react/icons/package.json

```diff
@@ -54,6 +54,8 @@
     ],
     "author": "Brightlayer UI <brightlayer-ui@eaton.com>",
     "license": "BSD-3-Clause",
+    "bugs": {
+        "url": "https://github.com/brightlayer-ui/progress-icons/issues",
```

Comment:
This line here is causing a problem; it's not a valid package.json.

`bugs: { ... } `

Reply:
Thanks, great catch!